src/engraving/libmscore/beam.cpp

```diff
@@ -47,6 +47,7 @@
 #include "system.h"
 #include "tremolo.h"
 #include "tuplet.h"
+#include "stemslash.h"
```

Comment:
I'd prefer to keep the #includes sorted alphabetically, just to keep it a bit tidy and avoid chaos. (Recently I did a PR to fix the sorting in basically all libmscore files, so it would be lovely to try to keep it clean from now on 🙂)

(Please forgive me for complaining about such a trivial useless thing)

Reply:
No worries, we all like tidiness ;)